packages/framework/src/Component/Transformers/ShowOnDomainBooleanToIntegerTransformer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Component\Transformers;
+
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Symfony\Component\Form\DataTransformerInterface;
+
+class ShowOnDomainBooleanToIntegerTransformer implements DataTransformerInterface
```

Comment:
This logic should be probably implemented right in the `DomainsType` - it is used in the three places only (Category, Payment, Transport). If it was just a collection of `CheckboxTypes`, the data would be in correct format without transformers.

Reply:
The DomainsType can be as simple as this, without the need of a transformer or any changes to current form types or theme:

```php
<?php

namespace Shopsys\FrameworkBundle\Form;

use Shopsys\FrameworkBundle\Component\Domain\Domain;
use Symfony\Component\Form\AbstractType;
use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
use Symfony\Component\Form\FormBuilderInterface;

class DomainsType extends AbstractType
{
    /**
     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
     */
    private $domain;

    /**
     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
     */
    public function __construct(Domain $domain)
    {
        $this->domain = $domain;
    }

    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        foreach ($this->domain->getAll() as $domainConfig) {
            $builder->add($domainConfig->getId(), CheckboxType::class, [
                'required' => false,
                'label' => $domainConfig->getName(),
            ]);
        }
    }
}

Reply:
great